src/components/kytos/inputs/Textarea.vue

```diff
@@ -51,6 +51,9 @@ export default {
     updateText(value){
       this.$emit('textarea', value)
```

Comment:
@rmotitsuki, thanks a lot for this patch, I managed to also use successfully both `action` callback and the `value` now syncs correctly. 

I wonder if we can get rid of these emit and action below, without them it also worked since the correct refs are being called again, do you think we can safely remove them, since this part is being touched here already?  

```
diff --git a/src/components/kytos/inputs/Textarea.vue b/src/components/kytos/inputs/Textarea.vue
index 35eb2e2..ccb7101 100644
--- a/src/components/kytos/inputs/Textarea.vue
+++ b/src/components/kytos/inputs/Textarea.vue
@@ -49,9 +49,6 @@ export default {
   },
   methods: {
     updateText(value){
-      this.$emit('textarea', value)
-      this.action(value)
-
       this.$emit('update:value', this.$refs.textarea.value)
       this.action(this.$refs.textarea.value)
     }
```

Reply:
I´m not sure. That´s why I left the code.